.openpublishing.redirection.json

```diff
@@ -1,5 +1,20 @@
 { 
   "redirections": [ 
+    {   
+        "source_path": "aspnetcore/fundamentals/repository-pattern.md", 
+        "redirect_url": "http://aka.ms/MicroservicesEbook", 
```

Comment:
Link to the exact section in the eBook that defines the Repository Pattern:
/dotnet/standard/microservices-architecture/microservice-ddd-cqrs-patterns/infrastructure-persistence-layer-design#the-repository-pattern